OBAKit/Helpers/OBACanvasView.m

```diff
@@ -0,0 +1,37 @@
+//
+//  OBACanvasView.m
+//  org.onebusaway.iphone
+//
+//  Created by Chad Royal on 10/10/16.
+//  Copyright © 2016 OneBusAway. All rights reserved.
+//
+
+#import "OBACanvasView.h"
+
+// This is a lightweight view derived from UIView to offer only an opportunity
+// to have a custom drawRect implementation with a reusable class rather than
+// having N tiny classes with different drawing code. Because there's no other
+// functionality needed, I chose a simple block model to do the drawing.
+
+#pragma mark - OBACanvasView
+
+@implementation OBACanvasView {
+    void(^_drawBlock)(CGRect);
```

Comment:
Any reason this isn't a property? I'm not sure it really makes a difference. Just curious about your decision.